sql-bricks.js

```diff
@@ -41,7 +41,7 @@
 
     var str = this.str;
     if (!opts)
-      opts = default_opts;
+      opts = _.extend({}, default_opts);

```

Comment:
@vitalif: SqlBricks is used occasionally in the browser (against WebSQL or against sqlite in cordova/phonegap, electron-shell, etc)... We probably will want to move to Object.assign() at some point, but I'm not sure if now's the time (but TBH I don't off the top of my head the adoption in different environments, etc)